packages/tanstack-start/src/utils/errors.ts

```diff
@@ -20,7 +20,7 @@ export const noFetchFnCtxPassedInGetAuth = createErrorMessage(`
 export const clerkHandlerNotConfigured = createErrorMessage(`
     It looks like you're trying to use Clerk without configuring the Clerk handler.
```

Comment:
If you're removing the indentation of the rest this should also be changed

```suggestion
It looks like you're trying to use Clerk without configuring the Clerk handler.
```